pkg/util/node/node.go

```diff
@@ -181,22 +181,21 @@ func GetNodeIP(client clientset.Interface, name string) net.IP {
 //   * "topology.kubernetes.io/zone"
 // GetZoneKey will first check failure-domain.beta.kubernetes.io/zone and if not exists, will then check
 // topology.kubernetes.io/zone
+// TODO(thockin): stop supporting "failure-domain.beta..." names after 1.20
 func GetZoneKey(node *v1.Node) string {
 	labels := node.Labels
 	if labels == nil {
 		return ""
 	}
 
-	// TODO: prefer stable labels for zone in v1.18
```

Comment:
Related to this TODO: https://github.com/kubernetes/kubernetes/pull/88493

Reply:
Will add a note.  After 20 goes, we can just nix it all